shared/src/Datadog.Trace.ClrProfiler.Native/dynamic_dispatcher.cpp

```diff
@@ -128,9 +124,9 @@ namespace datadog::shared::nativeloader
                     {
                         // Convert possible relative paths to absolute paths using the configuration file folder as base
                         // (current_path)
-                        std::string absoluteFilepathValue = fs::absolute(filepathValue).string();
+                        std::string absoluteFilepathValue = (configFolder / filepathValue).string();
                         Log::Debug("DynamicDispatcherImpl::LoadConfiguration: [", type, "] Loading: ", filepathValue, " [AbsolutePath=", absoluteFilepathValue,"] (", currentOsArch, ")" );
-                        if (fs::exists(absoluteFilepathValue))
+                        if (fs::exists(absoluteFilepathValue, ec))
```

Comment:
Is it worth logging the error code? 🤔

Reply:
👍 I think that would be interesting in case of a failure